content/rendering-with-batch/batch/monitor.md

```diff
@@ -61,9 +58,6 @@ You will need the appropriate program and video codecs to watch the mp4 generate
 Explore also the rest of the S3 folders and check the frames that were created.
 {{% /notice %}}
 
-
-## Monitoring
```

Comment:
If you delete this level 2 title you have to update the nested titles consequently making them level 2. I would refrain making this change since there are several titles related to monitoring and should be nested inside a monitoring section.